nss-day-cohort-47/tabloidfullstack-bingo
# Dcu create comment

# Description
User can create a comment on a post

## Type of change


# Testing Instructions
Check out dcu-create-comment branch, run the project, log in, click on a post's details button, click add comment, write in your comment. This comment should now show in the comment list for a post
# Checklist:
